src/Components/Aspire.Pomelo.EntityFrameworkCore.MySql/AspireEFMySqlExtensions.cs

```diff
@@ -68,6 +69,22 @@ public static partial class AspireEFMySqlExtensions
 
         builder.Services.AddDbContextPool<TContext>(ConfigureDbContext);
 
+        const string resilienceKey = "Microsoft.Extensions.Hosting.AspireEFMySqlExtensions.ServerVersion";
+        builder.Services.AddResiliencePipeline(resilienceKey, static builder =>
+        {
+            // Values are taken from MySqlRetryingExecutionStrategy.MaxRetryCount and MaxRetryDelay.
+            builder.AddRetry(new RetryStrategyOptions
+            {
+                ShouldHandle = static args => args.Outcome is { Exception: MySqlException { IsTransient: true } }
+                    ? PredicateResult.True()
+                    : PredicateResult.False(),
+                BackoffType = DelayBackoffType.Exponential,
+                MaxRetryAttempts = 6,
+                Delay = TimeSpan.FromSeconds(1),
+                MaxDelay = TimeSpan.FromSeconds(30),
```

Comment:
Why set `MaxDelay` while also setting `MaxRetryAttempts`? Isn't 1 always going to be shorter than the other?

Reply:
`MaxDelay` "Caps the calculated retry delay to a specified maximum duration". (So the delays might go 1, 2, 4, 8, 16, 30, 30, 30) instead of (...16, 32, 64, 128).

I suppose in practice, since `MaxRetryAttempts` is 6, this only caps the final retry from 32s to 30s.

This was "porting" the defaults from https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/blob/master/src/EFCore.MySql/MySqlRetryingExecutionStrategy.cs and https://github.com/dotnet/efcore/blob/main/src/EFCore/Storage/ExecutionStrategy.cs#L25-L37 for lack of any better defaults to arbitrarily choose; I'm not committed to these values and am happy to remove this line if it's more confusing than anything else.